src/main/java/io/confluent/examples/streams/PageViewRegionExample.java

```diff
@@ -196,7 +197,8 @@ public GenericRecord apply(final GenericRecord view, final String region) {
       })
       // count views by user, using hopping windows of size 5 minutes that advance every 1 minute
       .groupByKey() // no need to specify explicit serdes because the resulting key and value types match our default serde settings
-      .count(TimeWindows.of(5 * 60 * 1000L).advanceBy(60 * 1000L), "GeoPageViewsStore");
+      .windowedBy(TimeWindows.of(5 * 60 * 1000L).advanceBy(60 * 1000L))
```

Comment:
Nit: Use `TimeUnit` instead of magic numbers (like we do elsewhere)?

I am aware the original code wasn't using `TimeUnit` yet, either.